types/node/test/stream.ts

```diff
@@ -583,3 +583,17 @@ async function testTransferringStreamWithPostMessage() {
     // error TS2532: Cannot use 'stream' as a target of a postMessage call because it is not a Transferable.
     // port2.postMessage(stream, [stream]);
 }
+
+function testEndandDestroy() {
```

Comment:
Shouldn't this test function be in `node/v16/test/stream.ts`, since you're updating types for v16?